lib/ey-core/cli/deploy.rb

```diff
@@ -0,0 +1,65 @@
+class Ey::Core::Cli::Deploy < Ey::Core::Cli::Subcommand
+  title "deploy"
+  summary "Deploy your application"
+
+  option :environment, short: "e", long: "environment", description: "Name or id of the environment to deploy to.", argument: "Environment"
+  option :account,     short: 'c', long: 'account',     description: 'Name or ID of the account that the environment resides in.  If no account is specified, the app will deploy to the first environment that meets the criteria, in the accounts you have access to.', argument: 'Account name or id'
+  option :ref,         short: "r", long: "ref",         description: "A git reference to deploy.", argument: "ref"
+  option :migrate,     short: "m", long: "migrate",     description: "The migration command to run.  This option has a 50 character limit.", argument: "migrate"
+  option :app,         short: "a", long: "app",         description: "Application name or ID to deploy.  If :account is not specified, this will be the first app that matches the criteria in the accounts you have access to.", argument: "app"
+
+  switch :stream, long: "stream",     description: "Stream deploy output to this console."
+  switch :verbose, short: "v", long: "verbose",    description: "Stream deploy output to this console."
```

Comment:
oh, its supposed to just be an alias to stream for backwards compatibility but forgot to check the switch